<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewKeyValue</class>
 <widget class="QWidget" name="NewKeyValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New (key, value) data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="keyEditField"/>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="valueEditField"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="valueTypeLabel">
     <property name="text">
      <string>Value type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="valueTypeComboBox">
     <item>
      <property name="text">
       <string>Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Integer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="addKeyValueButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
